pihole

```diff
@@ -90,6 +95,7 @@ function helpFunc {
 	echo ":::  -s, setupLCD				Automatically configures the Pi to use the 2.8 LCD screen to display stats on it"
 	echo ":::  -c, chronometer			Calculates stats and displays to an LCD"
 	echo ":::  -h, help				Show this help dialog"
+	echo ":::  -v, version          Show current versions"
```

Comment:
I think the spacing is off. (though it might not look different in the terminal unless tabs are different than 4 spaces)